static/js/play.js

```diff
@@ -35,6 +36,11 @@ let app = new Vue({
 
         home: function (event) {
             window.location.replace("/");
+        },
+
+        startNow: function (event) {
+            app.$data.started = true;
```

Comment:
I know other places in this file use `app.$data`, but you can also use `this` instead, which is more concise